scripts/upgrade_samples.sh

```diff
@@ -61,36 +61,32 @@ fi
 for DEPENDENCIES_FILE in `find . -type f -iname "dependencies.kt"` ; do
     COMPOSE_BLOCK=false;
     ACCOMPANIST_BLOCK=false;
-    KOTLIN_BLOCK=false;
     MADE_CHANGE=false;
     TEMP_FILENAME="${DEPENDENCIES_FILE}_new";
     while IFS= read -r line; do
         if [[ $line == *"val version ="* && "$compose_version" != "" ]] && $COMPOSE_BLOCK = true; then
             echo "$line" | sed -En 's/".*"/"'$compose_version'"/p'
             MADE_CHANGE=true;
-        elif [[ $line == *"val snapshot ="* ]] && $COMPOSE_BLOCK = true; then
+        elif [[ $line == *"val snapshot ="* && "$snapshot_version" != "" ]] && $COMPOSE_BLOCK = true; then
```

Comment:
Allowing a blank snapshot_version is done on purpose so you can clear it out when switching back to a normal version